x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -192,6 +193,38 @@ public void testRolloverAction() throws Exception {
         assertBusy(() -> assertTrue(indexExists(originalIndex)));
     }
 
+    public void testRolloverAlreadyExists() throws Exception {
+        String originalIndex = index + "-000001";
+        String secondIndex = index + "-000002";
+        createIndexWithSettings(originalIndex, Settings.builder().put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
+            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0)
+            .put(RolloverAction.LIFECYCLE_ROLLOVER_ALIAS, "alias"));
+
+        // create policy
+        createNewSingletonPolicy("hot", new RolloverAction(null, null, 1L));
+        // update policy on index
+        updatePolicy(originalIndex, policy);
+
+        // Manually create the new index
+        Request request = new Request("PUT", "/" + secondIndex);
+        request.setJsonEntity("{\n \"settings\": " + Strings.toString(Settings.builder().put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
+            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0).build()) + "}");
+        client().performRequest(request);
+        // wait for the shards to initialize
+        ensureGreen(secondIndex);
+
+        // index another doc to trigger the policy
+        index(client(), originalIndex, "_id", "foo", "bar");
+
+        assertBusy(() -> {
+            logger.info(originalIndex + ": " + getStepKeyForIndex(originalIndex));
+            logger.info(secondIndex + ": " + getStepKeyForIndex(secondIndex));
+            assertThat(getStepKeyForIndex(originalIndex), equalTo(new StepKey("hot", RolloverAction.NAME, ErrorStep.NAME)));
+            assertThat(getFailedStepForIndex(originalIndex), equalTo("update-rollover-lifecycle-date"));
+            assertThat(getReasonForIndex(originalIndex), equalTo("index [" + originalIndex + "] has not rolled over yet"));
```

Comment:
I agree that this error message is confusing. Just to check I understand correctly. The problem with the second index is actually that it doesn't have the rollover alias setting set and doesn't have any rollover info?

Reply:
Here's an example: If you have `test-000001` and manually create `test-000002` before rollover fires (and don't do anything else), then `test-000001` will end up in the error state with this message because `test-000001` has not been rolled over, and therefore doesn't have any attached `RolloverInfo`.  `test-000001` is still the target of the alias and still has the rollover alias setting.

If `test-000002` had been manually created and the alias had been manually switched to point to `test-000002`, `attempt_rollover` would fail due to https://github.com/elastic/elasticsearch/pull/35065

Reply:
I think we should just change the message in this step to say something like:
```
No rollover info found for index [INDEX_NAME]. Either the index has not yet rolled over or a subsequent index was created outside of Index Lifecycle Management. 
```